cmd/huker.go

```diff
@@ -211,6 +211,13 @@ func main() {
 				printUsageAndExit()
 			}
 		}
+
+		if _, err := os.Stat(pkgRoot); err != nil {
+			if os.IsNotExist(err) {
```

Comment:
Nice founding.  there's a ./lib directory under my desktop,  so did not found this.  BTW, when pkgRoot directory does not exist , I prefer to mkdir it  or just say that "the  ./lib directory does not exist, please create the directory firstly. " , rather than log the back trace to user..

Reply:
For demo or testing, considering easy to run,  it's ok to make lib directory by default. But for the production environment,  it costs large disk io for downloading payload from an agent. So if "lib" locates at the same disk as the huker process,  they will compete with each other for the disk io.  I think it's important to prepare the pkgRoot directory carefully, panic is more suitable.

Reply:
>I think it's important to prepare the pkgRoot directory carefully, panic is more suitable.
Yes, it's.  I mean print the hint "the ./lib directory does not exist, please create the directory firstly.", and then exit with return code 1 directly. no need to log the stack trace to user, besides, the traces are all go path from the system os package . not huker  package.